react-design/my-app/src/RotatingBanner.jsx

```diff
@@ -1,16 +1,19 @@
+import { useState } from 'react';
 import Indicators from './Indicators';
 import PreviousButton from './PreviousButton';
 import NextButton from './NextButton';
 import Banner from './Banner';
 import './App.css';
 
 export default function RotatingBanner({ listItems }) {
+  const [currentIndex /*, setCurrentIndex */] = useState(2);
```

Comment:
No need to comment out this! If you're creating a state, it is assumed you'll be updating it at some point!